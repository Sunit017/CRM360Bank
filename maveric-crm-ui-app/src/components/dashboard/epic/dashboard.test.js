import Dashboard from "..";
import renderWithRedux from "../../../testUtil";
import { cleanup, fireEvent, screen } from "@testing-library/react";

afterEach(cleanup);

test("Render dashboard page", async () => {
  const container = renderWithRedux(<Dashboard />, {});
  expect(container).toMatchSnapshot();
});

test("Dashboard page icon click ", () => {
  const container = renderWithRedux(<Dashboard />, {});
  fireEvent.click(screen.getByTestId("customer360Search-button"));
});
