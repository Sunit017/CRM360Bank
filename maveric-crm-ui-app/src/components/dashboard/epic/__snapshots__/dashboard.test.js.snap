// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render dashboard page 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="sc-bczRLJ bgHaRI"
      >
        <div
          class="css-2lzdb8-MuiStack-root"
        >
          <div
            class="MuiBox-root css-it15o"
          >
            <div
              class="MuiBox-root css-1vnp42r"
            >
              <div
                class="sc-dkzDqf cPfyTI"
                style="margin-bottom: 10px;"
              >
                Account 360
              </div>
              <form
                style="display: flex;"
              >
                <div
                  class="MuiFormControl-root MuiTextField-root css-r3a8lr-MuiFormControl-root-MuiTextField-root"
                >
                  <div
                    class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                  >
                    <input
                      aria-invalid="false"
                      autocomplete="off"
                      class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                      id=":r0:"
                      name="accountNumber"
                      placeholder="Search by Account Number (Allows only digit)"
                      type="text"
                      value=""
                    />
                    <fieldset
                      aria-hidden="true"
                      class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                    >
                      <legend
                        class="css-hdw1oc"
                      >
                        <span
                          class="notranslate"
                        >
                          â€‹
                        </span>
                      </legend>
                    </fieldset>
                  </div>
                </div>
                <button
                  class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                  style="padding: 2px 0px;"
                  tabindex="0"
                  type="submit"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1ha5wk2-MuiSvgIcon-root"
                    data-testid="SearchIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                    />
                  </svg>
                  <span
                    class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                  />
                </button>
              </form>
            </div>
            <div
              class="MuiBox-root css-1hd9puk"
            >
              <button
                class="sc-hKMtZM dOIFkn"
                data-testid="customer360Search-button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                  data-testid="PersonSearchIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <circle
                    cx="10"
                    cy="8"
                    r="4"
                  />
                  <path
                    d="M10.35 14.01C7.62 13.91 2 15.27 2 18v2h9.54c-2.47-2.76-1.23-5.89-1.19-5.99zm9.08 4.01c.36-.59.57-1.28.57-2.02 0-2.21-1.79-4-4-4s-4 1.79-4 4 1.79 4 4 4c.74 0 1.43-.22 2.02-.57L20.59 22 22 20.59l-2.57-2.57zM16 18c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z"
                  />
                </svg>
                <div
                  class="sc-dkzDqf cPfyTI"
                >
                  Customer 360 Search
                </div>
              </button>
            </div>
            <div
              class="MuiBox-root css-0"
            >
              <div
                class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-spacing-md-3 MuiGrid-spacing-lg-1 css-vobqoz-MuiGrid-root"
                style="text-align: center;"
              >
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="SupportAgentIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M21 12.22C21 6.73 16.74 3 12 3c-4.69 0-9 3.65-9 9.28-.6.34-1 .98-1 1.72v2c0 1.1.9 2 2 2h1v-6.1c0-3.87 3.13-7 7-7s7 3.13 7 7V19h-8v2h8c1.1 0 2-.9 2-2v-1.22c.59-.31 1-.92 1-1.64v-2.3c0-.7-.41-1.31-1-1.62z"
                      />
                      <circle
                        cx="9"
                        cy="13"
                        r="1"
                      />
                      <circle
                        cx="15"
                        cy="13"
                        r="1"
                      />
                      <path
                        d="M18 11.03C17.52 8.18 15.04 6 12.05 6c-3.03 0-6.29 2.51-6.03 6.45 2.47-1.01 4.33-3.21 4.86-5.89 1.31 2.63 4 4.44 7.12 4.47z"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      Case 360
                    </div>
                  </button>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="HowToRegIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="m9 17 3-2.94c-.39-.04-.68-.06-1-.06-2.67 0-8 1.34-8 4v2h9l-3-3zm2-5c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4m4.47 8.5L12 17l1.4-1.41 2.07 2.08 5.13-5.17 1.4 1.41z"
                        fill-rule="evenodd"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      OPS 360
                    </div>
                  </button>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="AssessmentIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      Metrics and Reporting
                    </div>
                  </button>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="NotificationsActiveRoundedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M18 16v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.68-1.5-1.51-1.5S10.5 3.17 10.5 4v.68C7.63 5.36 6 7.92 6 11v5l-1.3 1.29c-.63.63-.19 1.71.7 1.71h13.17c.89 0 1.34-1.08.71-1.71L18 16zm-6.01 6c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zM6.77 4.73c.42-.38.43-1.03.03-1.43-.38-.38-1-.39-1.39-.02C3.7 4.84 2.52 6.96 2.14 9.34c-.09.61.38 1.16 1 1.16.48 0 .9-.35.98-.83.3-1.94 1.26-3.67 2.65-4.94zM18.6 3.28c-.4-.37-1.02-.36-1.4.02-.4.4-.38 1.04.03 1.42 1.38 1.27 2.35 3 2.65 4.94.07.48.49.83.98.83.61 0 1.09-.55.99-1.16-.38-2.37-1.55-4.48-3.25-6.05z"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      Notification
                    </div>
                  </button>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="TaskRoundedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="m19.41 7.41-4.83-4.83c-.37-.37-.88-.58-1.41-.58H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8.83c0-.53-.21-1.04-.59-1.42zm-9.18 9.88-2.12-2.12a.9959.9959 0 0 1 0-1.41c.39-.39 1.02-.39 1.41 0l1.41 1.41 3.54-3.54c.39-.39 1.02-.39 1.41 0 .39.39.39 1.02 0 1.41l-4.24 4.24c-.38.4-1.02.4-1.41.01zM14 9c-.55 0-1-.45-1-1V3.5L18.5 9H14z"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      Task Creation
                    </div>
                  </button>
                </div>
                <div
                  class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
                >
                  <button
                    class="sc-hKMtZM dOIFkn"
                  >
                    <svg
                      aria-hidden="true"
                      class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                      data-testid="BadgeRoundedIcon"
                      focusable="false"
                      viewBox="0 0 24 24"
                    >
                      <path
                        d="M20 7h-5V4c0-1.1-.9-2-2-2h-2c-1.1 0-2 .9-2 2v3H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V9c0-1.1-.9-2-2-2zM9 12c.83 0 1.5.67 1.5 1.5S9.83 15 9 15s-1.5-.67-1.5-1.5S8.17 12 9 12zm3 6H6v-.43c0-.6.36-1.15.92-1.39.64-.28 1.34-.43 2.08-.43s1.44.15 2.08.43c.55.24.92.78.92 1.39V18zm1-9h-2V4h2v5zm4.25 7.5h-2.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h2.5c.41 0 .75.34.75.75s-.34.75-.75.75zm0-3h-2.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h2.5c.41 0 .75.34.75.75s-.34.75-.75.75z"
                      />
                    </svg>
                    <div
                      class="sc-dkzDqf cPfyTI"
                    >
                      Roles and access
                    </div>
                  </button>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="sc-bczRLJ bgHaRI"
    >
      <div
        class="css-2lzdb8-MuiStack-root"
      >
        <div
          class="MuiBox-root css-it15o"
        >
          <div
            class="MuiBox-root css-1vnp42r"
          >
            <div
              class="sc-dkzDqf cPfyTI"
              style="margin-bottom: 10px;"
            >
              Account 360
            </div>
            <form
              style="display: flex;"
            >
              <div
                class="MuiFormControl-root MuiTextField-root css-r3a8lr-MuiFormControl-root-MuiTextField-root"
              >
                <div
                  class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
                >
                  <input
                    aria-invalid="false"
                    autocomplete="off"
                    class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
                    id=":r0:"
                    name="accountNumber"
                    placeholder="Search by Account Number (Allows only digit)"
                    type="text"
                    value=""
                  />
                  <fieldset
                    aria-hidden="true"
                    class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
                  >
                    <legend
                      class="css-hdw1oc"
                    >
                      <span
                        class="notranslate"
                      >
                        â€‹
                      </span>
                    </legend>
                  </fieldset>
                </div>
              </div>
              <button
                class="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-textPrimary MuiButton-sizeMedium MuiButton-textSizeMedium css-1e6y48t-MuiButtonBase-root-MuiButton-root"
                style="padding: 2px 0px;"
                tabindex="0"
                type="submit"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1ha5wk2-MuiSvgIcon-root"
                  data-testid="SearchIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.5 14h-.79l-.28-.27C15.41 12.59 16 11.11 16 9.5 16 5.91 13.09 3 9.5 3S3 5.91 3 9.5 5.91 16 9.5 16c1.61 0 3.09-.59 4.23-1.57l.27.28v.79l5 4.99L20.49 19l-4.99-5zm-6 0C7.01 14 5 11.99 5 9.5S7.01 5 9.5 5 14 7.01 14 9.5 11.99 14 9.5 14z"
                  />
                </svg>
                <span
                  class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
                />
              </button>
            </form>
          </div>
          <div
            class="MuiBox-root css-1hd9puk"
          >
            <button
              class="sc-hKMtZM dOIFkn"
              data-testid="customer360Search-button"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                data-testid="PersonSearchIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <circle
                  cx="10"
                  cy="8"
                  r="4"
                />
                <path
                  d="M10.35 14.01C7.62 13.91 2 15.27 2 18v2h9.54c-2.47-2.76-1.23-5.89-1.19-5.99zm9.08 4.01c.36-.59.57-1.28.57-2.02 0-2.21-1.79-4-4-4s-4 1.79-4 4 1.79 4 4 4c.74 0 1.43-.22 2.02-.57L20.59 22 22 20.59l-2.57-2.57zM16 18c-1.1 0-2-.9-2-2s.9-2 2-2 2 .9 2 2-.9 2-2 2z"
                />
              </svg>
              <div
                class="sc-dkzDqf cPfyTI"
              >
                Customer 360 Search
              </div>
            </button>
          </div>
          <div
            class="MuiBox-root css-0"
          >
            <div
              class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 MuiGrid-spacing-md-3 MuiGrid-spacing-lg-1 css-vobqoz-MuiGrid-root"
              style="text-align: center;"
            >
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="SupportAgentIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M21 12.22C21 6.73 16.74 3 12 3c-4.69 0-9 3.65-9 9.28-.6.34-1 .98-1 1.72v2c0 1.1.9 2 2 2h1v-6.1c0-3.87 3.13-7 7-7s7 3.13 7 7V19h-8v2h8c1.1 0 2-.9 2-2v-1.22c.59-.31 1-.92 1-1.64v-2.3c0-.7-.41-1.31-1-1.62z"
                    />
                    <circle
                      cx="9"
                      cy="13"
                      r="1"
                    />
                    <circle
                      cx="15"
                      cy="13"
                      r="1"
                    />
                    <path
                      d="M18 11.03C17.52 8.18 15.04 6 12.05 6c-3.03 0-6.29 2.51-6.03 6.45 2.47-1.01 4.33-3.21 4.86-5.89 1.31 2.63 4 4.44 7.12 4.47z"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    Case 360
                  </div>
                </button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="HowToRegIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="m9 17 3-2.94c-.39-.04-.68-.06-1-.06-2.67 0-8 1.34-8 4v2h9l-3-3zm2-5c2.21 0 4-1.79 4-4s-1.79-4-4-4-4 1.79-4 4 1.79 4 4 4m4.47 8.5L12 17l1.4-1.41 2.07 2.08 5.13-5.17 1.4 1.41z"
                      fill-rule="evenodd"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    OPS 360
                  </div>
                </button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="AssessmentIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M19 3H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2zM9 17H7v-7h2v7zm4 0h-2V7h2v10zm4 0h-2v-4h2v4z"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    Metrics and Reporting
                  </div>
                </button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="NotificationsActiveRoundedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M18 16v-5c0-3.07-1.64-5.64-4.5-6.32V4c0-.83-.68-1.5-1.51-1.5S10.5 3.17 10.5 4v.68C7.63 5.36 6 7.92 6 11v5l-1.3 1.29c-.63.63-.19 1.71.7 1.71h13.17c.89 0 1.34-1.08.71-1.71L18 16zm-6.01 6c1.1 0 2-.9 2-2h-4c0 1.1.89 2 2 2zM6.77 4.73c.42-.38.43-1.03.03-1.43-.38-.38-1-.39-1.39-.02C3.7 4.84 2.52 6.96 2.14 9.34c-.09.61.38 1.16 1 1.16.48 0 .9-.35.98-.83.3-1.94 1.26-3.67 2.65-4.94zM18.6 3.28c-.4-.37-1.02-.36-1.4.02-.4.4-.38 1.04.03 1.42 1.38 1.27 2.35 3 2.65 4.94.07.48.49.83.98.83.61 0 1.09-.55.99-1.16-.38-2.37-1.55-4.48-3.25-6.05z"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    Notification
                  </div>
                </button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="TaskRoundedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="m19.41 7.41-4.83-4.83c-.37-.37-.88-.58-1.41-.58H6c-1.1 0-1.99.9-1.99 2L4 20c0 1.1.89 2 1.99 2H18c1.1 0 2-.9 2-2V8.83c0-.53-.21-1.04-.59-1.42zm-9.18 9.88-2.12-2.12a.9959.9959 0 0 1 0-1.41c.39-.39 1.02-.39 1.41 0l1.41 1.41 3.54-3.54c.39-.39 1.02-.39 1.41 0 .39.39.39 1.02 0 1.41l-4.24 4.24c-.38.4-1.02.4-1.41.01zM14 9c-.55 0-1-.45-1-1V3.5L18.5 9H14z"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    Task Creation
                  </div>
                </button>
              </div>
              <div
                class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-2 css-1urn9ow-MuiGrid-root"
              >
                <button
                  class="sc-hKMtZM dOIFkn"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1dxcdol-MuiSvgIcon-root"
                    data-testid="BadgeRoundedIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M20 7h-5V4c0-1.1-.9-2-2-2h-2c-1.1 0-2 .9-2 2v3H4c-1.1 0-2 .9-2 2v11c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V9c0-1.1-.9-2-2-2zM9 12c.83 0 1.5.67 1.5 1.5S9.83 15 9 15s-1.5-.67-1.5-1.5S8.17 12 9 12zm3 6H6v-.43c0-.6.36-1.15.92-1.39.64-.28 1.34-.43 2.08-.43s1.44.15 2.08.43c.55.24.92.78.92 1.39V18zm1-9h-2V4h2v5zm4.25 7.5h-2.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h2.5c.41 0 .75.34.75.75s-.34.75-.75.75zm0-3h-2.5c-.41 0-.75-.34-.75-.75s.34-.75.75-.75h2.5c.41 0 .75.34.75.75s-.34.75-.75.75z"
                    />
                  </svg>
                  <div
                    class="sc-dkzDqf cPfyTI"
                  >
                    Roles and access
                  </div>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "store": Object {
    "@@observable": [Function],
    "dispatch": [Function],
    "getState": [Function],
    "replaceReducer": [Function],
    "subscribe": [Function],
  },
  "unmount": [Function],
}
`;
